File: Python/RSAEncryption/math_operation/euclidGCD.py

# Function to find the GCD (Greatest Common Divisor) OR in indonesian => FPB (Faktor Persekutuan Terbesar)
# using euclid algorithm
# 
# Parameter m and n position is doesn't matter
# has return value of the GCD result in number format like INT
def euclidGCD(m, n):
    while n != 0:
        temp = m % n
        m = n
        n = temp

    return m

File: PHP/RSAEncryption/lib/math/euclidGCD.php
/*
    HANDLE GCD Operation using Euclid Algorithm

    In Indonesia => Faktor Persekutuan Terbesar
*/

<?php
    function euclidGCD($m, $n){
        $temp = 0;
        
        while ($n != 0) {
            $temp = $m % $n;
            $m = $n;
            $n = $temp;
        }

        return $m;
    }
?>